### PR TITLE
Nz/3491 key person form

### LINKED ISSUE #3491: [Feature] Update Key State Personnel
**Description and related issues -**

Hide alert message for now, it will be addressed further in #3348

---

Display description from inside the form above the Add Primary Contact button
"Personnel listed in this section must be assigned to APD by name, role, time commitment, and cost, where applicable. Add at least one point of contact (key state personnel)." 
Only necessary on the Primary APD Point of Contact

Once Primary POC is added, then it should look like this

---

Add in-line validation to Key State Personnel form. The validation should appear after the user hits the Add (previously Done) button. If there are validation errors, the errors should display, nothing should be saved, and it should stay on the form until the errors are fixed.

---

Update the view of added Primary APD Point of Contact

---

Update the view of added Addition Key Personnel

### This task is done when…
- [x] hide alert message when list is empty
- [x] update description at the top of the form
- [x] add in-line validation
- [x] update review to match Figma

### PR DESCRIPTION
**Pull request by Sun-Mountain**
Resolves #3491 

**Description-**
Hide alert message for now, it will be addressed further in #3348

**This pull request changes...**

-  hide alert message when list is empty
- update description at the top of the form
- add in-line validation
- update review to match Figma

**This pull request also touches…**

- The form validations/saving forms

**Steps to manually verify this change...**

1. Cannot add primary contact or key personnel if missing text.
2. Can submit primary contact or key personnel with all text areas filled out.

### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [x] Code has been reviewed by someone other than the original author
- [x] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by tbolt — APPROVED** [review]
Code looks good 👍

**Comment by mirano-darren** [comment]
Works for me as well. Well done! :D

**Review by thetif — APPROVED** [review]

File: integrationTests/cypress/helpers/apd/key-state-personnel.js [hunk 1]
```diff
@@ -75,121 +75,122 @@ export const testKeyStatePersonnelWithData = years => {
...
 
     cy.fixture('users').then(userData => {
+      const firstUser = userData[0];
+      const secondUser = userData[1];
+      const thirdUser = userData[2];
+
       cy.get('input[name="apd-state-profile-mdname"]')
         .clear()
-        .type(userData[0].name);
+        .type(firstUser.name);
```
[the remark this answers is not in the document]
Comment (Sun-Mountain) [reply integrationTests/cypress/helpers/apd/key-state-personnel.js:84]: Ah thanks, for some reason I thought I had copied all of your settings. Maybe that was on my old laptop.

**Review by thetif — APPROVED** [review]
code looks good

**Comment by thetif** [comment]
The error message doesn't go away when you fill out the field.
Screen Shot 2022-03-03 at 4 32 22 PM

**Comment by thetif** [comment]
Also I'm able to save even if the fields aren't filled out.

File: web/src/containers/activity/OutcomesAndMetrics/OutcomeAndMetricForm.js [hunk 2]
```diff
@@ -7,65 +7,60 @@ import { connect } from 'react-redux';
 import Icon, { faPlusCircle } from '../../../components/Icons';
 import Review from '../../../components/Review';
 
+<<<<<<< HEAD
```
Comment (tbolt) [inline web/src/containers/activity/OutcomesAndMetrics/OutcomeAndMetricForm.js:10]: looks like this merge needs to be cleaned up

**Review by tbolt — APPROVED** [review]
code looks good 👍 take a look at OutcomeAndMetricForm though, there's still some merge conflict comments in there.

**Pull request closed without merging**